components/cards/card.css

```diff
@@ -0,0 +1,241 @@
+}
+
+img{
+    height: 26rem;
+    width: 36rem;
+}
+
+.hero_action_icon{
+    margin-left: auto;
+}
+.hero__action{
+    display: flex;
+    align-items: flex-start;
+    margin-right: 55px;
+}
+.hero__btns{
+    display: flex;
+    align-items: flex-start;
+    /* margin: 1px 0px; */
+}
+.btn__margin{
+    margin: 12px;
+    background: none;
+    border:none;
+    color: var(--card-text);
+}
+.btn__action:hover{
+    border-radius: 13px;
+    background-color: var(--card-hover);
+    cursor: pointer;
+}
+.hero__title{
+    margin-right: 18rem;
+    font-weight: 500;
+    color: var(--card-text);
+}
+.hero__caption{
+    margin: 13px 13px;
+    font-size: 1.3rem;
+}
+.btn__icon{
+    background: none;
+    border: none;
+    margin-right: 0.9rem;
+    color: var(--card-text);
+}
+.btn__icon:hover{
+    cursor: pointer;
+}
+.hero:hover{
+    cursor: pointer;
+}
+small{
+    color:var(--card-text);
+}
+/* 
+text on img
+*/
+.hero__c1{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 32rem;
+    width: 30rem;
+    margin: 4rem;
+    border: var(--card-text);
+    border-radius: 8px;
+    background-color: var(--card-bg-color);
+    overflow: hidden;
+    box-shadow: 2px 2px 6px 3px darkgrey;
+}
+.hero__text__overlay{
+    position: absolute;
+    margin-right: 16rem;
+    margin-top: 6rem;
+    font-weight: 500;
+    color: var(--card-text);
+}
+.img__c1{
+    position: relative;
+    height: 22rem;
+    width: 30rem;
+}
+/* 
+HORIZONTAL CARD/ WITH BADGE
+*/
+.card-img{
+    position: relative;
+    width: 14rem;
+    height: 17.5rem;
+    border-radius: 8px;
+}
+.badge{
+    position: absolute;
+    display: flex;
+    width: 14rem;
+    height: 1rem;
+    background-color: var(--card-badge);
+    top: 37.3%;
+    text-align: center;
+    color: var(--card-text);
+    justify-content: center;
+    align-items: center;
+}
+.btn{
+    
+    padding: 0.5rem ;
+    margin: 1.25rem 0;
+    background-color: transparent;
+    border: 2px solid grey;
+    font-size: 1rem;
+    text-align: center;
+    width: 12rem;
+    cursor: pointer;
+    font-weight: 600;
+}
+.btn-solid{
+    background-color: var(--card-btn);
+}
+.horizontal-card{
+    display: flex;
+    width: 100%;
+    background-color: var(--card-hr-bg);
+    color: var(--card-text);
+    align-items: center;
+    justify-content: center;
+    padding: 2rem 12rem;
+}
+
+.card-text{
+    padding: 3rem;
+    width: 33%;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+.tag{
+    background-color: var(--card-tag);
+    color: var(--card-text);
+    padding: 3px 4px;
+    border-radius: 6px;
+}
+.price{
+    display: flex;
+    gap: 10px;
+    align-items: center;
+    justify-content: flex-start;
+}
+.line{
+    text-decoration:line-through;
+    opacity: 30%;
+}
+.bold{
+    font-weight: 600;
+}
+i{
+    margin: 8px;
+}
+
+/* 
+vertical card
+*/
+.card-end{
+    display: flex;
```

Comment:
You have used the property of display:flex;justify-content:center;align-items:center at multiple places so you can make an utility class of it and attach it to the respective element and then the rest of the properties you can add up.

Reply:
Ok done